styles.css

```diff
@@ -215,4 +215,81 @@
   .bottom-border {
     border-bottom: 1px solid #dfe1e6;
   }
+
+  #contact {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 100px 24px 0 24px;
+    background-image: url("./assets/shape-background.png");
```

Comment:
- [ ] One of the objectives of this project is to understand how to parse a design template precisely. Kindly update the background image to match the design. 🤝️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/220605909-12fd5b24-14a7-4376-8c3a-09e792346191.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/220605929-80d02669-5a3d-40c3-88fd-b6df77e9ef0c.png" width="300">|